src/ElectricPokemon.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon{
+    public ElectricPokemon(String type, String name, int level, int hp, String food, String sound) {
+        super(type, name, level, hp, food, sound);
```

Comment:
Het type kun je hier al invullen, dat is namelijk "electric". dan hoef je het type ook niet meer te vragen in de parameters